internal/command/container.go

```diff
@@ -239,16 +239,16 @@ func fyneRelease(ctx Context, image containerImage) error {
 	case androidOS:
 		workDir = volume.JoinPathContainer(workDir, ctx.Package)
 		if ctx.Keystore != "" {
-			args = append(args, "-keyStore", ctx.Keystore)
```

Comment:
These also need a version check.

Reply:
Fixed, and improved the version checks on the way.